libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorMediaUploadListener.java

```diff
@@ -3,8 +3,8 @@
 import org.wordpress.android.util.helpers.MediaFile;
 
 public interface EditorMediaUploadListener {
-    void onMediaUploadSucceeded(String localId, MediaFile mediaFile);
+    void onMediaUploadSucceeded(String localId, EditorFragmentAbstract.MediaType mediaType, MediaFile mediaFile);
```

Comment:
Adding an extra param here for the `MediaType` seems redundant since we're already getting that from the `MediaFile`, which we're also sending.

Could we make `getEditorMimeType()` a (static) method of `EditorFragmentAbstract` instead, and use that from `AztecEditorFragment` and from `EditPostActivity`? Then we can drop this parameter (we'll still need it in `onMediaUploadFailed` below though, I guess).